package.json

```diff
@@ -22,7 +22,7 @@
     "npm": ">=4.0"
   },
   "dependencies": {
-    "aws-sdk": "^2.268.1",
+    "aws-sdk": "github:jpb/aws-sdk-js#e6a1b24910e9182bb2e03c111d71b4c95198149b",
```

Comment:
Can we clone this fork to `github:unbounce/aws-sdk-js`? I assume this will be a temporary thing and we can remove it once upstream merges your PR.

Reply:
I've done that and fleshed out the commit message a bit. LGTM and works well.